.travis.yml

```diff
@@ -1,89 +1,149 @@
 language: objective-c
-osx_image: xcode9.3beta
+osx_image: xcode10.2
 gemfile: Gemfile
 
```

Comment:
let's add:

```
addons:
  homebrew:
    packages:
    - swiftlint
    update: true
```

and we can remove the swiftlint stuff below

Reply:
Oh awesome. Yeah totally